src/ray/gcs/redis_context.cc

```diff
@@ -506,6 +564,14 @@ Status RedisContext::Connect(const std::string &address,
   // Ray has some restrictions for RedisDB. Validate it here.
   ValidateRedisDB(*this);
```

Comment:
If it's Redis Sentinel, what will `INFO CLUSTER` returns?

Reply:
It will return empty - basically that section doesn't exist. Here's a full result of `INFO` on redis sentinel:
```
# Server
redis_version:7.0.7
redis_git_sha1:00000000
redis_git_dirty:0
redis_build_id:5dad631ce1b0fc10
redis_mode:sentinel
os:Linux 6.8.0-1017-aws x86_64
arch_bits:64
monotonic_clock:POSIX clock_gettime
multiplexing_api:epoll
atomicvar_api:c11-builtin
gcc_version:9.4.0
process_id:7
process_supervised:no
run_id:de8c925350decfbb0abd5940bf718855eb191f8d
tcp_port:26379
server_time_usec:1730346466295003
uptime_in_seconds:728236
uptime_in_days:8
hz:14
configured_hz:10
lru_clock:2293218
executable:/redis-sentinel
config_file:/etc/redis/sentinel.conf
io_threads_active:0

# Clients
connected_clients:3
cluster_connections:0
maxclients:10000
client_recent_max_input_buffer:20480
client_recent_max_output_buffer:0
blocked_clients:0
tracking_clients:0
clients_in_timeout_table:0

# Stats
total_connections_received:121374
total_commands_processed:2334908
instantaneous_ops_per_sec:3
total_net_input_bytes:135052866
total_net_output_bytes:163753996
total_net_repl_input_bytes:0
total_net_repl_output_bytes:0
instantaneous_input_kbps:0.17
instantaneous_output_kbps:0.02
instantaneous_input_repl_kbps:0.00
instantaneous_output_repl_kbps:0.00
rejected_connections:0
sync_full:0
sync_partial_ok:0
sync_partial_err:0
expired_keys:0
expired_stale_perc:0.00
expired_time_cap_reached_count:0
expire_cycle_cpu_milliseconds:12828
evicted_keys:0
evicted_clients:0
total_eviction_exceeded_time:0
current_eviction_exceeded_time:0
keyspace_hits:0
keyspace_misses:0
pubsub_channels:0
pubsub_patterns:0
pubsubshard_channels:0
latest_fork_usec:0
total_forks:0
migrate_cached_sockets:0
slave_expires_tracked_keys:0
active_defrag_hits:0
active_defrag_misses:0
active_defrag_key_hits:0
active_defrag_key_misses:0
total_active_defrag_time:0
current_active_defrag_time:0
tracking_total_keys:0
tracking_total_items:0
tracking_total_prefixes:0
unexpected_error_replies:0
total_error_replies:97084
dump_payload_sanitizations:0
total_reads_processed:2502068
total_writes_processed:2380696
io_threaded_reads_processed:0
io_threaded_writes_processed:0
reply_buffer_shrinks:29
reply_buffer_expands:0

# CPU
used_cpu_sys:1282.061897
used_cpu_user:931.763629
used_cpu_sys_children:0.032320
used_cpu_user_children:0.026019
used_cpu_sys_main_thread:1282.061999
used_cpu_user_main_thread:931.762826

# Sentinel
sentinel_masters:1
sentinel_tilt:0
sentinel_tilt_since_seconds:-1
sentinel_running_scripts:0
sentinel_scripts_queue_length:0
sentinel_simulate_failure_flags:0
master0:name=redis-ha,status=ok,address=10.112.187.90:6379,slaves=2,sentinels=3
```